plugins/source/gcp/codegen/domains.go

```diff
@@ -19,13 +17,6 @@ var domainsResources = []*Resource{
 		UnimplementedServer: &pb.UnimplementedDomainsServer{},
 		RequestStructFields: `Parent: fmt.Sprintf("projects/%s/locations/-", c.ProjectId),`,
 		Imports:             []string{"fmt"},
-		OverrideColumns: []codegen.ColumnDefinition{
```

Comment:
`Registration` doesn't have a self link field so adding `OverrideColumns` didn't have an impact of the generated code